src/components/pathway/PathwayModal.jsx

```diff
@@ -0,0 +1,200 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Skeleton from 'react-loading-skeleton';
+import { AppContext } from '@edx/frontend-platform/react';
+import {
+  Icon, MarketingModal, ModalDialog, Container, Row, Col, Collapsible, Button, Image,
+} from '@edx/paragon';
+import { Assignment, BookOpen, VerifiedBadge } from '@edx/paragon/icons';
+import { useLearnerPathwayData } from './data/hooks';
+
+const coursesAndProgramsText = pathway => {
```

Comment:
nit: what about moving this to utils?

Reply:
I don't see any usage of this outside the pathway modal so I think for now we can leave it here. We can move this to utils if need arises in future?